### PR TITLE
ADD: Add intake catalog

### PR DESCRIPTION
**Pull request by mgrover1**
Add an intake catalog of AWS cloud hosted datasets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,78 @@ These files are used as sample data in xwrf testing suites, tutorials, and docum
...
+
+We also include an [intake](https://github.com/intake/intake) catalog, which makes use of [intake-xarray](https://github.com/intake/intake-xarray).
+
+You can access this data catalog using the following syntax:
+
+```python
+import intake
+
+# Read in the data catalog from github
+cat = intake.open_catalog('https://raw.githubusercontent.com/mgrover1/sample-wrf-data/main/catalog.yml')
```
Comment (jthielen) [inline README.md:33]:
```suggestion
cat = intake.open_catalog('https://raw.githubusercontent.com/xarray-contrib/xwrf-data/main/catalogs/catalog.yml')
```

File: README.md [hunk 2]
```diff
@@ -19,3 +19,78 @@ These files are used as sample data in xwrf testing suites, tutorials, and docum
...
+
+# Read one of the sample datasets
+ds = cat["xwrf-sample-ssp245"].to_dask()
+
+ds
```
Comment (jthielen) [inline README.md:38]: I'd suggest to split the python code and output into separate blocks

File: README.md [hunk 3]
```diff
@@ -19,3 +19,78 @@ These files are used as sample data in xwrf testing suites, tutorials, and docum
...
+    WEST-EAST_PATCH_START_UNSTAG:    1
+    W_DAMPING:                       0
+    YSU_TOPDOWN_PBLMIX:              0
+```
+
+This catalog includes:
+
+- [`xwrf-sample-ssp245`](./catalogs/xwrf-sample-ssp245.json): Custom WRF output file for testing purposes
+- [`ssp245_gcm_wrfout_combined`](./catalogs/ssp245_gcm_wrfout_combined.json): Custom WRF output file for testing purposes
+- [`xwrf-sample-ssp585`](./catalogs/xwrf-sample-ssp585.json): Custom WRF output file for testing purposes
```
Comment (jthielen) [inline README.md:96]:
```suggestion
- [`xwrf-sample-ssp245`](./catalogs/ssp245_gcm_wrfout_combined.json): WRF Downscaled CMIP6; SSP245; Combined virtual dataset (2099-10, d01)
- [`xwrf-sample-ssp585`](./catalogs/ssp585_gcm_wrfout_combined.json): WRF Downscaled CMIP6; SSP585; Combined virtual dataset (2099-10, d01)
- [`single-output-sample`](./catalogs/gcm_wrfout_d01_2099-10-01_00_00_00.json): Virtual WRF output dataset for testing purposes, consisting of a single time from xwrf-sample-ssp585 (2099-10-01T00:00)
```

File: README.md [hunk 4]
```diff
@@ -19,3 +19,78 @@ These files are used as sample data in xwrf testing suites, tutorials, and docum
...
+    ...                              ...
+    WEST-EAST_PATCH_END_STAG:        271
+    WEST-EAST_PATCH_END_UNSTAG:      270
+    WEST-EAST_PATCH_START_STAG:      1
+    WEST-EAST_PATCH_START_UNSTAG:    1
+    W_DAMPING:                       0
+    YSU_TOPDOWN_PBLMIX:              0
+```
+
+This catalog includes:
```
Comment (jthielen) [inline README.md:92]:
```suggestion
This catalog includes the following virtual datasets created using [kerchunk](https://fsspec.github.io/kerchunk/):
```

**Review by jthielen — CHANGES_REQUESTED** [review]
Thanks for adding these, and the README additions are a wonderful idea! Just a few suggestions to clean up the README documentation:

**Comment by jthielen** [comment]
Also, @xarray-contrib/xwrf-devs, we'll need someone with write access to approve this to activate CI.

**Comment by andersy005** [comment]
> Also, https://github.com/orgs/xarray-contrib/teams/xwrf-devs, we'll need someone with write access here to approve this to activate CI.

I just updated the permissions, let me know if you don't have write access

**Comment by jthielen** [comment]
> I just updated the permissions, let me know if you don't have write access

Thanks! I now have write access.

File: README.md (hunk 2 above)
Comment (mgrover1) [reply README.md:38]: Ahh - good point! Can do!

**Review by jthielen — APPROVED** [review]

**Comment by jthielen** [comment]
I updated the combined json files after fixing my kerchunk approach to combine properly over the `Time` dimension. Merging now so that we can test on the `main` branch URLs, so if there are any further fixed needed here, a follow-up PR would be best!

**Pull request merged**